helpers/extension/driver.js

```diff
@@ -151,7 +151,7 @@ class ExtensionProtocol extends ChromeProtocol {
           return reject(chrome.runtime.lastError);
         }
         this._debuggerConnected = true;
-
+        this.handleUnexpectedDetach();
```

Comment:
Wonder if we should just add this to the constructor. As it is we add it on attach, but we never remove it when detaching. Feels lopsided. 👈